supervisely/api/project_api.py

```diff
@@ -1202,3 +1204,32 @@ def check_imageset_backup(self, id: int) -> Optional[Dict]:
         response = self._api.get("projects.images.get-backup-archive", {ApiField.ID: id})
 
         return response.json()
+
+    def append_classes(self, id: int, classes: Union[List[ObjClass], ObjClassCollection]) -> None:
+        """
+        Append new classes to given Project.
+
+        :param id: Project ID in Supervisely.
+        :type id: int
+        :param classes: New classes
+        :type classes: :class: ObjClassCollection or List[ObjClass]
+        :return: None
+        :rtype: :class:`NoneType`
+        :Usage example:
+
+         .. code-block:: python
+
+            import supervisely as sly
+
+            os.environ['SERVER_ADDRESS'] = 'https://app.supervise.ly'
+            os.environ['API_TOKEN'] = 'Your Supervisely API Token'
+            api = sly.Api.from_env()
+
+            proj_id = 28145
+            lung_obj_class = sly.ObjClass("lung", sly.Mask3D)
+            api.project.append_classes(proj_id, [lung_obj_class])
+        """
+        meta_json = self.get_meta(id)
+        meta = ProjectMeta.from_json(meta_json)
+        meta = meta.add_obj_classes(classes)
```

Comment:
test? ObjClassCollection

Reply:
`ObjClassCollection` works
In `KeyIndexedCollection.add_items()` we have `[*ObjClassCollection]` to extract ObjClass objects in list